lib/rubocop/formatter/formatter_set.rb

```diff
@@ -73,7 +76,7 @@ def add_formatter(formatter_type, output_path = nil)
           output = $stdout
         end
 
-        self << formatter_class.new(output)
+        self << formatter_class.new(output, @options)
```

Comment:
But if a custom formatter overrides that with `def initialize(output)`, you still get a `wrong number of arguments (2 for 1)` error from this call.